commcare_connect/program/models.py

```diff
@@ -18,3 +18,9 @@ def save(self, *args, **kwargs):
         if not self.id:
             self.slug = slugify_uniquely(self.name, self.__class__)
         super().save(*args, **kwargs)
+
+
+class ManagedOpportunity(Opportunity):
```

Comment:
I think we need to override the organization column as well to make it nullable, since these will not have organziations at first.